tests/ui/impl-trait/impl-fn-hrtb-bounds.stderr

```diff
@@ -10,7 +10,7 @@ help: consider using the `'static` lifetime, but this is uncommon unless you're
 LL | fn d() -> impl Fn() -> (impl Debug + 'static) {
    |                                      ~~~~~~~
 
-error: higher kinded lifetime bounds on nested opaque types are not supported yet
+error[E0657]: higher kinded lifetime bounds on nested opaque types are not supported yet
```

Comment:
Docs for E0657 kind of are vague... I could rewrite them.

Reply:
While we're changing this, these are really higher ranked not higher kinded, no?

Reply:
Yeah, you're right